client-app/shared/payment/components/payment-processing-skyflow.vue

```diff
@@ -0,0 +1,233 @@
+    },
+  ];
+};
+
+const { t } = useI18n();
+const notifications = useNotifications();
+
+const loading = ref(false);
+const ga = useGoogleAnalytics();
+
+interface IProps {
+  order: CustomerOrderType;
+  disabled?: boolean;
+}
+
+const initialized = ref(false);
+
+let skyflowClient: Skyflow, skyflowTableName: string, composableContainer: ComposableContainer;
+
+async function initPayment() {
+  try {
+    const { publicParameters, errorMessage } = await initializePayment({
+      orderId: props.order.id,
+      paymentId: props.order.inPayments[0]!.id,
+    });
+
+    if (errorMessage) {
+      showErrorNotification();
+      console.error(errorMessage);
+      return;
+    }
+
+    if (!publicParameters) {
+      showErrorNotification();
+      console.error("Skyflow module parameters are not provided");
+      return;
+    }
+
+    skyflowTableName = getParameter(publicParameters, "tableName");
+
+    skyflowClient = Skyflow.init({
+      vaultID: getParameter(publicParameters, "vaultID"),
+      vaultURL: getParameter(publicParameters, "vaultURL"),
+      getBearerToken: () => Promise.resolve(getParameter(publicParameters, "accessToken")),
+      options: {
+        logLevel: Skyflow.LogLevel.ERROR,
+        env: Skyflow.Env.DEV,
+      },
+    });
+
+    initialized.value = true;
+  } catch (e) {
+    showErrorNotification();
+    console.error(e);
+  }
+}
+async function pay() {
+  loading.value = true;
+
+  const res = (await composableContainer.collect({
+    tokens: true,
+  })) as SkyflowResponseType;
+
+  console.log("skyflow response", res);
+
+  if (!res?.records) {
+    showErrorNotification();
+    console.error("Skyflow data saving error");
+  }
+
+  const newRes = await authorizePayment({
+    orderId: props.order.id,
+    paymentId: props.order.inPayments[0]!.id,
+    parameters: objectToKeyValue(res.records.find((el) => el.fields)?.fields as FieldsType),
+  });
+
+  console.log("XAPI authorizePayment response", newRes);
+
+  ga.purchase(props.order);
+
+  loading.value = false;
+}
+
+function showErrorNotification() {
+  notifications.error({
+    duration: 5000,
+    single: true,
+    text: t("shared.payment.skyflow.errors.payment_failed"),
+  });
+}
+
+onMounted(async () => {
+  await initPayment();
+  createForm();
+});
+
+function createForm() {
+  const containerOptions = {
+    layout: [1, 1, 2],
+    styles: {
+      base: {
+        border: "1px solid #eae8ee",
+        padding: "10px 16px",
+        borderRadius: "4px",
+        margin: "12px 4px",
+      },
+    },
+    errorTextStyles: {
+      base: {
+        color: "red",
+      },
+    },
+  };
+
+  const container = skyflowClient.container(Skyflow.ContainerType.COMPOSABLE, containerOptions) as ComposableContainer;
+
+  const collectStylesOptions = {
+    inputStyles: {
+      base: {
+        fontFamily: "Inter",
+        fontStyle: "normal",
+        fontWeight: 400,
+        fontSize: "14px",
+        lineHeight: "21px",
+        width: "200px",
```

Comment:
We need to find a way to get these values from our css, otherwise it's not customizable (i.e. will have different color palette than configured and don't reflect changes)